forge-gui/res/cardsfolder/upcoming/sage_of_the_unknowable.txt

```diff
@@ -0,0 +1,7 @@
+Name:Sage of the Unknowable
+ManaCost:2

```

Comment:
need to confirm cost

Reply:
I know, this one got into the upload by mistake.